atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/SnapshotTransactionTest.java

```diff
@@ -1715,10 +1715,10 @@ public void getSortedColumnsThrowsIfLockIsLostMidway() {
 
         // should still be able to get all but last element of the elements for the first batch;
         // next batch is preemptively fetched when last element of curr batch is retrieved
-        List<Cell> retrievedEntries = IntStream.range(1, batchHint - 1)
+        List<Cell> retrievedEntries = IntStream.range(1, batchHint)
```

Comment:
Mind explaining this change?

Reply:
I think this is going to get undone actually.

Weirdly, these two specific tests fail locally but pass on circle (with batchHint - 1); and they pass locally but fail on circle with the current setup. Maybe a me problem if they agree with circle locally for you?